### PR TITLE
Update fg-delegate.md

### PR DESCRIPTION
**Pull request by kat-enos**
DEL relnotes for February 1, 2023 projected SaaS release

# Harness Developer Pull Request
Thanks for helping us make the Developer Hub better. The PR will be looked at
by the maintainers. 

## What Type of PR is This?

- [ ] Issue
- [ ] Feature
- [ x ] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [ ] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

**Review by pratmit — DISMISSED** [review]

File: docs/first-gen/firstgen-release-notes/fg-delegate.md [hunk 1]
```diff
@@ -11,6 +11,50 @@ To identify the cluster that hosts your account, open Harness FirstGen and go to
...
+
+Harness FirstGen release 783xx includes the following feature changes and fixes for Harness Delegate.
+
+### Security enhancements
+
+This release introduces the following security enhancements:
+
+- The immutable delegate image was refreshed with updated versions of client tools. This reduces security vulnerabilities for the delegate and enhances security. The following table details the updates. (DEL-5688)
+  
+  | **Third-party tool** | **781xx and earlier** | **783xx and later** |
```
Comment (michaelcretzman) [inline docs/first-gen/firstgen-release-notes/fg-delegate.md:25]: Interesting list seeing as we have been saying these are the versions we ship with for 6 months:

https://developer.harness.io/docs/getting-started/supported-platforms-and-technologies#sdks-installed-with-the-delegate

Not sure what's going on here, but the RN looks fine.

**Review by michaelcretzman — DISMISSED** [review]
approving

File: docs/first-gen/firstgen-release-notes/fg-delegate.md [hunk 2]
```diff
@@ -11,6 +11,50 @@ To identify the cluster that hosts your account, open Harness FirstGen and go to
 For FirstGen SaaS release notes, see [Harness SaaS Release Notes (FirstGen)](/docs/first-gen/firstgen-release-notes/harness-saa-s-release-notes.md). For Self-Managed Enterprise Edition release notes, see [Self-Managed Enterprise Edition (FirstGen)](/docs/first-gen/firstgen-release-notes/harness-on-prem-release-notes.md).
 
 
+## February 1, 2023, version 783xx
```
Comment (kat-enos) [inline docs/first-gen/firstgen-release-notes/fg-delegate.md:14]:
```suggestion
## February 6, 2023, version 78321
```

File: docs/first-gen/firstgen-release-notes/fg-delegate.md [hunk 3]
```diff
@@ -11,6 +11,50 @@ To identify the cluster that hosts your account, open Harness FirstGen and go to
...
+### Delegate version 783xx
```
Comment (kat-enos) [inline docs/first-gen/firstgen-release-notes/fg-delegate.md:15]:
```suggestion
### Delegate version 78306
```

File: docs/first-gen/firstgen-release-notes/fg-delegate.md [hunk 4]
```diff
@@ -11,6 +11,50 @@ To identify the cluster that hosts your account, open Harness FirstGen and go to
...
+
+Harness FirstGen release 783xx includes the following feature changes and fixes for Harness Delegate.
```
Comment (kat-enos) [inline docs/first-gen/firstgen-release-notes/fg-delegate.md:17]:
```suggestion
Harness FirstGen release 78321 includes the following feature changes and fixes for Harness Delegate.
```

File: docs/first-gen/firstgen-release-notes/fg-delegate.md (hunk 1 above)
Comment (kat-enos) [inline docs/first-gen/firstgen-release-notes/fg-delegate.md:25]:
```suggestion
  | **Third-party tool** | **78101 and earlier** | **78306 and later** |
```

**Review by douglas-j-bothwell — APPROVED** [review]

**Pull request merged**